### ISSUE TITLE
cosine used in base_train

### ISSUE THREAD

**Issue by Atlantis-me** [JungHunOh/CLOSER_ECCV2024#1]
hi, in helper.py line 66 and 69, you used symbol cosine, but i didn't find where the cosine is defined, can you help me?

**Comment by JungHunOh** [comment]
Hi. Thanks for your interest in our work.

Sorry for the error in the code.

Could you replace 'cosine' with 'logits' and try again?

We will check this issue and re-upload the code as soon as possible.

**Comment by Atlantis-me** [comment]
that helps, thanks a lot!
